### DISCUSSION TITLE
Traducción al ucraniano

### DISCUSSION THREAD

**Discussion by lllPapilll** [CharlieFuu69/RenPy_RhythmBeats#9] [General]
1.0

**Comment by lllPapilll** [comment]
[ukr.zip](https://github.com/CharlieFuu69/RenPy_RhythmBeats/files/11127580/ukr.zip)

**Reply by CharlieFuu69** [reply]
Los scripts están en proceso de revisión. Confirmaré su adición al juego cuando finalice la revisión.

**Reply by CharlieFuu69** [reply]
Script aprobado. Estoy empezando a implementarlo, aunque estoy viendo que tendré que utilizar una fuente de texto alternativa a la que el juego tiene por defecto.
